exampleSite/content/fragments/repositories/docs.md

```diff
@@ -0,0 +1,32 @@
++++
+fragment = "content"
+weight = 300
+
+title = "Documentation"
+
+[sidebar]
+  sticky = true
++++
+
+### Variables
+
+#### user
+*type: string*
+
+The username of the user you want the repositories of.
+
+#### count
+*type: number*
+*default: 1000*
+
+The number of repositories that will be rendered.
+
+#### sort
+*type: number*
+*default: updated_at*
+
+The variable name to use for sorting repositories. 
```

Comment:
Why "variable name"?

Reply:
I can't think of a good explanation, not happy with it either. Any suggestions?